src/account/account.repository.knex.ts

```diff
@@ -59,10 +59,24 @@ export class KnexAccountRepository {
                 ap_outbox_url: draft.apOutbox?.href ?? null,
                 ap_following_url: draft.apFollowing?.href ?? null,
                 ap_liked_url: draft.apLiked?.href ?? null,
-                ap_public_key: JSON.stringify(
-                    await exportJwk(draft.apPublicKey),
-                ),
-                ap_private_key: draft.apPrivateKey
+                ap_public_key: (draft as any).dualKeyPairs
+                    ? JSON.stringify({
+                          keys: await Promise.all(
```

Comment:
I don't love the idea of storing two different data types in the same column here.

I think we need a discussion about either:

1. Migrating all columns to store multiple keys
2. Adding new columns for the new keys